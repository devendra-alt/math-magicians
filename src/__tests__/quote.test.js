import { MemoryRouter } from "react-router-dom";
import  renderer from "react-test-renderer" ;
import Quote from "../components/quote.js";
import React from "react";

describe('renders correctly', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
      <MemoryRouter>
        < Quote/>
      </MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });