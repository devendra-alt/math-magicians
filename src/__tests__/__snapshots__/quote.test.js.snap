// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <nav
      className="nav-bar"
    >
      <h1
        className="title"
      >
        Math Magicians
      </h1>
      <ul
        className="nav-links"
      >
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/quotes"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <article
    className="quote-container"
  >
    <p
      className="quote"
    />
    <p
      className="quote-author"
    />
    <button
      className="quote-suffler"
      onClick={[Function]}
      type="submit"
    >
      <span
        className="quote-suffler-tooltip"
      >
        reload
      </span>
    </button>
  </article>,
]
`;
